mlir/include/mlir/Dialect/OpenACC/OpenACCOps.td

```diff
@@ -1642,7 +1667,8 @@ def OpenACC_LoopOp : OpenACC_Op<"loop",
   let hasCustomAssemblyFormat = 1;
   let assemblyFormat = [{
     oilist(
-        `gang` `` custom<GangClause>($gangOperands, type($gangOperands),
+        `combined` `(` custom<CombinedConstructs>($combined) `)`
```

Comment:
Same comment about oilist

Reply:
As far as I can tell, oilist works well enough here - "combined" indeed appears first if it is present.

Reply:
For printing yes but I think we want to enforce that this must appear just after the op name when parsing.

Reply:
I see what you mean. OK.

Reply:
Done